addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -1309,28 +1308,41 @@ registry.WebsiteAnimate = publicWidget.Widget.extend({
      * @param {Element} el
      */
     _scrollWebsiteAnimate(el) {
-        const scroll = $(el).scrollTop();
-        // Handle reverse scrolling
-        const direction = (scroll < this.lastScroll) ? -1 : 1;
-        this.lastScroll = scroll;
-
         _.each(this.$target.find('.o_animate:not(.o_animate_in_dropdown)'), el => {
             const $el = $(el);
-            const elHeight = $el.height();
-            const elOffset = direction * Math.max((elHeight * this.offsetRatio), this.offsetMin);
             const state = $el.css("animation-play-state");
+            let visible = false;
+
+            const positionOfEl = el.getClientRects()[0];
```

Comment:
By doing this you have actually changed the way animations are played too much...
The problem is that with this change, the position of the element to animate (relative to the viewport) is the position of the element with its "transform" state and no longer the "real" position of the element in the DOM. (the position when the animation is finished). It's easy to test it with the "Fade-in Down", now the animation is started too early when scrolling down the page.

I haven't gone further in the review but maybe it just needs something specific for the "slide out" footer as suggested in the spec of the task..
(Also remember to keep in progress PR on draft ;) )